apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/controller/DailyRatesController.java

```diff
@@ -0,0 +1,52 @@
+package com.apmc.apmcSpringBoot.controller;
+
+
+import com.apmc.apmcSpringBoot.Exception.Response;
+import com.apmc.apmcSpringBoot.Exception.ResponseException;
+import com.apmc.apmcSpringBoot.model.DailyRates;
+import com.apmc.apmcSpringBoot.service.DailyRateService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+@RestController
+@RequestMapping("/dailyRates")
+public class DailyRatesController {
+
+    @Autowired
+    private DailyRateService dailyRateService;
+
+    @GetMapping("")
+    public List<DailyRates> getDailyRatesByDate(@RequestParam String day) throws ParseException {
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
If this 'dateFormat' is needed frequently, put this as a static final in a 'constant' file